server/server.go

```diff
@@ -34,15 +37,20 @@ func NewServer(config *ServerConfig, db models.DB, auth *providers.Auth, schema
 }
 
 func (s *Server) Start() {
+	config := s.config
 	h := handler.New(&handler.Config{
 		Schema:   s.schema,
 		Pretty:   true,
 		GraphiQL: true,
 	})
 
-	log.Println("Listening on http://localhost:8000")
-	http.Handle("/graphql", enableCorsMiddleware(s.VerifyTokenMiddleware(gqlMiddleware(h))))
+	if config.GraphiQLEnabled {
```

Comment:
We shouldn't need this check here (see comment above)

Reply:
Ah yes that make sense.